test/unit/lib/plugins/aws/package/compile/events/api-gateway/lib/rest-api.test.js

```diff
@@ -148,6 +151,30 @@ describe('#compileRestApi()', () => {
       Properties: {
         BinaryMediaTypes: ['*/*'],
         DisableExecuteApiEndpoint: undefined,
+        Description: undefined,
+        EndpointConfiguration: {
```

Comment:
is there a reason why this needed to be updated too? (the fields other than `description`)

Reply:
hmm... I changed only description.
Maybe Github is confusing to show difference with near new test case(which is removed on newer commit) ...?